api/models/app.go

```diff
@@ -82,6 +86,19 @@ func GetApp(name string) (*App, error) {
 
 var regexValidAppName = regexp.MustCompile(`\A[a-zA-Z][-a-zA-Z0-9]{3,29}\z`)
 
+func (a *App) StackName() string {
+	if a.Tags == nil {
+		return shortNameToStackName(a.Name)
+	}
+
+	nameTag, ok := a.Tags["Name"]
```

Comment:
Oh sure. Heh I only changed it so I could use `nameTag` later... but... `a.Name`... (doh)! I'll fix up.